test/parallel/test-internal-fs-syncwritestream.js

```diff
@@ -66,6 +66,30 @@ const filename = tmpdir.resolve('sync-write-stream.txt');
   assert.strictEqual(stream.fd, null);
 }
 
+// Verify that the callback will be called when already destroy()ed.
+{
+  const fd = fs.openSync(filename, 'w');
+  const stream = new SyncWriteStream(fd);
+  const theErr = new Error('my error');
+  const cb = () => {};
+
+  stream.on('close', common.mustCall());
+  assert.strictEqual(stream.destroy(), stream);
+  stream.destroy(theErr, common.mustCall(cb));
+  stream.destroySoon(theErr, common.mustCall(cb));
+  stream._destroy(theErr, common.mustCall(cb));
+}
+
+// Verify that the file is not closed when autoClose=false
+{
+  const fd = fs.openSync(filename, 'w');
+  const stream = new SyncWriteStream(fd, { autoClose: false });
+
+  stream.on('close', common.mustNotCall());
+  stream._destroy(null, () => {});
```

Comment:
Ditto.

Reply:
btw, is this "Ditto" means, "no callback here" ? or "no `_destroy()` call directly" ?

Reply:
The latter.

Reply:
But I thought that it is okay to using and testing inner function (`_write(), _destroy()`, etc.) directly, in light of the subtest block at the bottom of this tests, that is using `_write()`
Putting this aside, as I mentioned above, I think there is no way to test effectiveness of `autoClose` without using `_destroy()` directly.
When the `stream.destroy()` called, the file will be set closed and `stream.fd` will be null, regardless of `autoClose` value, Then, I wonder that we even need the `autoClose` option at all.
Which direction do you think as good?

Reply:
You can do this

```js
{
  const fd = fs.openSync(filename, 'w');
  const stream = new SyncWriteStream(fd, { autoClose: false });

  stream.destroy();

  // This throws an error if the file descriptor was closed.
  fs.fstatSync(fd);
}
```

Reply:
wow thank you for your help!